src/data/periods.js

```diff
@@ -0,0 +1,26 @@
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Date Periods. This is here to accomodate the split build process. The typescript copy should supersede this when appropriate.
+ */
+const Periods = [
+	{
+		label: __( '7 days', 'altis' ),
+		value: 'P7D',
+		diff: 'P7D',
+		number: 7,
+	},
+	{
+		label: __( '30 days', 'altis' ),
+		value: 'P30D',
+		diff: 'P30D',
+		number: 30,
+	},
+	{
+		label: __( '90 days', 'altis' ),
+		value: 'P90D',
+		diff: null,
+		number: 90,
+	},
+];
+export default Periods;
```

Comment:
1. I created this for the periods to work with the date range. I added `number` as the number value is needed for the request for the posts. It wasn't accepting `P7D` as a value only `7`. 

2. What is the purpose of the `diff` value as I wasn't able to identify where/what it's used for?

Reply:
1. I was suggesting updating the API to accept values like `P7D` rather than just numbers so we were able to do this and standardise on one approach
2. `diff` is used to indicate the time range to get data for to calculate the percentage change comparison, e.g. when showing data for the last 7 days, compare it to the 7 days before that.